interfaces/apparmor/template.go

```diff
@@ -233,13 +233,15 @@ var defaultTemplate = []byte(`
 
   # Writable home area for this version.
   owner @{HOME}/snap/@{SNAP_NAME}/@{SNAP_REVISION}/** wl,
+  owner @{HOME}/snap/@{SNAP_NAME}/shared/** wl,
```

Comment:
Quickly discussed this with the team and the response was positive.

So let's go ahead with the proposal, just replacing the "shared" term by "common" everywhere please.